contrib/fleece-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+SET(LIBRARY_DIR ${ClickHouse_SOURCE_DIR}/contrib/fleece)
+
+add_library(fleece
+${LIBRARY_DIR}/Fleece/Array.cc
+${LIBRARY_DIR}/Fleece/Array.hh
+${LIBRARY_DIR}/Fleece/Benchmark.hh
+${LIBRARY_DIR}/Fleece/Dict.cc
+${LIBRARY_DIR}/Fleece/Dict.hh
+${LIBRARY_DIR}/Fleece/Encoder.cc
+${LIBRARY_DIR}/Fleece/Encoder.hh
+${LIBRARY_DIR}/Fleece/Endian.hh
+${LIBRARY_DIR}/Fleece/Fleece+CoreFoundation.h
+${LIBRARY_DIR}/Fleece/Fleece.h
+${LIBRARY_DIR}/Fleece/Fleece.hh
+${LIBRARY_DIR}/Fleece/Fleece.pch
+${LIBRARY_DIR}/Fleece/Fleece_C_impl.cc
+${LIBRARY_DIR}/Fleece/Fleece_C_impl.hh
+${LIBRARY_DIR}/Fleece/FleeceException.cc
+${LIBRARY_DIR}/Fleece/FleeceException.hh
+${LIBRARY_DIR}/Fleece/forestdb_endian.h
+${LIBRARY_DIR}/Fleece/Internal.hh
+${LIBRARY_DIR}/Fleece/JSON5.cc
+${LIBRARY_DIR}/Fleece/JSON5.hh
+${LIBRARY_DIR}/Fleece/JSONConverter.cc
+${LIBRARY_DIR}/Fleece/JSONConverter.hh
+${LIBRARY_DIR}/Fleece/JSONEncoder.cc
```

Comment:
Not very sure but it seems it can be built without `JSONEncoder.*` files here as well as the dependency on `vendor/jsonsl` below because you've written your own `JSONEncoder` which uses `RapidJSON`.

Reply:
I will try to remove it.